manage_issues.py

```diff
@@ -0,0 +1,579 @@
+        try:
+            if int(issue_type):
+                __ensure_project_issue_fields(args.project, issue_type)
+                print(
+                    "Project",
+                    args.project,
+                    "issue_type",
+                    ISSUE_TYPES_TO_NAME[args.project][issue_type],
+                )
+                json.dump(
+                    ISSUE_TYPE_FIELDS[args.project][issue_type], sys.stdout, indent=2
+                )
+                print("")
+        except ValueError:
+            pass
+
+
+def dump(args):
+    """Dump an issue."""
+    for issue_id in args.params:
+        issue = jira.issue(issue_id)
+        json.dump(issue.raw, sys.stdout, indent=2)
+
+
+# for simple fields, map the name used when giving
+# an argument to the name used with the Jira api
+# fields not listed here require special handling
+# or a different api (e.g. status is a transition)
+ARGS_TO_JIRA_FIELDS = {
+    "project": "project",
+    "issue_type": "issuetype",
+    "summary": "summary",
+    "epic_name": "Epic Name",
+    "description": "description",
+    "component": "components",
+    "version": "fixVersions",
+    "itm": "Internal Target Milestone",
+    "dtm": "Dev Target Milestone",
+    "dev_ack": "ACKs Check",
+    "story_points": "Story Points",
+    "sprint": "Sprint",
+    "doc_text_type": "Release Note Type",
+    "doc_text": "Release Note Text",
+    "docs_impact": "Product Documentation Required",
+    "product": "Products",
+}
+
+create_issues = []
+
+
+@click.command()
+@click.option("--component", type=str, help="component")
+@click.option(
+    "--github-url",
+    type=str,
+    help="https url of github issue or pr used to create or update jira issue",
+)
+@click.option(
+    "--version",
+    type=str,
+    help="Internal Target Release e.g. rhel-9.6",
+)
+@click.option(
+    "--itm",
+    type=int,
+    help="Internal Target Milestone",
+)
+@click.option(
+    "--dtm",
+    type=int,
+    help="Dev Target Milestone",
+)
+@click.option(
+    "--dev-ack",
+    type=str,
+    help="Give dev ack",
+)
+@click.option(
+    "--status",
+    type=str,
+    help="issue status",
+)
+@click.option(
+    "--project",
+    type=str,
+    required=True,
+    help="Project for issues",
+)
+@click.option(
+    "--issue-type",
+    type=click.Choice(["Bug", "Story", "Task", "Epic"], case_sensitive=False),
+    help="Type of issue",
+)
+@click.option(
+    "--summary",
+    type=str,
+    help="Issue summary - short title for issue",
+)
+@click.option(
+    "--epic-name",
+    type=str,
+    help="Epic name - required for epics",
+)
+@click.option(
+    "--description",
+    type=str,
+    help="Issue description - long, multiline information about issue",
+)
+@click.option(
+    "--role",
+    multiple=True,
+    help="One or more role names",
+)
+@click.option(
+    "--debug",
+    is_flag=True,
+    help="Turn on debug logging.",
+)
+@click.option(
+    "--doc-text-type",
+    type=str,
```

Comment:
Can script specify available types like for summary like in `type=click.Choice(["Bug", "Story", "Task", "Epic"], case_sensitive=False),`

Reply:
yes